content/applications/general/email_communication/email_servers.rst

```diff
@@ -1,222 +1,212 @@
+
+A G Suite server can be used for any Odoo hosting type. To do so, set up the SMTP relay service.
+The configuration steps are explained in `Google documentation <https://support.google.com
+/a/answer/2956491?hl=en>`__.
 
 Restriction
 -----------
-Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try using
-465, 587, or 2525.
+
+Please note that port 25 is blocked for security reasons on the SaaS and Odoo.sh platforms. Try
+using ports 465, 587, or 2525 instead.
 
 .. _email_communication/default_from:
 
 Use a default "From" email address
 ----------------------------------
 
-Sometimes, an email's "From" (outgoing) address can belong to a 
-different domain, and that can be a problem.
-
-For example, if a customer with address *mary@customer.example.com* responds to
-a message, Odoo will try to redistribute that same email to other subscribers
-in the thread. But if the domain *customer.example.com* forbids that kind of
-usage for security, the Odoo's redistributed email would get
-rejected by some recipients' mail servers.
-
-To avoid those kind of problems, you should make sure all emails use a "From"
-address from your authorized domain.
+Sometimes, an email's "From" (outgoing) address can belong to a different domain, and that can be a
+problem.
 
-If your MTA supports `SRS (Sender Rewriting Scheme)
-<https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme>`_, you can enable it
-to handle these situations. However, that is more complex and requires more
-technical knowledge that is not meant to be covered by this documentation.
+For example, if a customer with the email address *mary\@customer.example.com* responds to a
+message, Odoo will try to redistribute that same email to the other subscribers in the thread.
+However, if the domain *customer.example.com* forbids that kind of usage for security, the email
+that Odoo is trying to redistribute would get rejected by some recipients' email servers.
 
-Instead, you can also configure Odoo to do something similar by itself:
+To avoid that problem, make sure all emails use a "From" address from the same authorized domain.
 
-#.  Set your domain name in the General Settings.
+If the MTA supports `SRS (Sender Rewriting Scheme) <https://en.wikipedia.org/wiki/Sender_Rewriting
+_Scheme>`_, SRS can be enabled to handle these situations. However, that is more complex and
+requires more technical knowledge that is not meant to be covered by this documentation.
 
-    .. image:: email_servers/alias-domain.png
-       :align: center
+Instead, Odoo can be configured to do something similar by itself:
 
-#.  In developer mode, go to :menuselection:`Settings --> Technical -->
-    Parameters --> System Parameters`.
+#. Set the :guilabel:`Alias Domain` name in the :menuselection:`Settings --> General Settings
+   --> Discuss`.
 
-#.  Add one system parameter from these:
+   .. image:: email_servers/alias-domain.png
+      :align: left
+      :width: 100%
```

Comment:
Sorry, I didn't make myself clear, to align left, you must drop the align center, and that's it. The `align: left` option is not supported. :sweat_smile: 
```suggestion
```
-->
```rst
.. image:: email_servers/alias-domain.png
   :alt: Setting the domain alias configuration on Odoo.
```